scrapy/pipelines/files.py

```diff
@@ -13,6 +13,7 @@
 from six.moves.urllib.parse import urlparse
 from collections import defaultdict
 import six
+import mimetypes
```

Comment:
:lipstick: Could you insert it in alphabetical order (I know not all entries here do that :disappointed:)

Reply:
Didn't notice that :D